sympy/matrices/tests/test_commonmatrix.py

```diff
@@ -674,6 +674,20 @@ def test_multiplication():
         assert c[1, 0] == 3*5
         assert c[1, 1] == 0
 
+def test_matmul():
+    a = Matrix([[1, 2], [3, 4]])
+
+    #Check a@2 case
+    assert a.__matmul__(2) == NotImplemented
+
+    #To check 2@a case
+    try:
+        eval('2 @ a')
```

Comment:
Add a comment here that this is done this way because @ is only supported in Python 3.5+.